tests/laser/test_transaction.py

```diff
@@ -8,7 +8,7 @@
 
 def test_intercontract_call():
     # Arrange
-    cfg.gbl_next_uid = 0
+    cfg.new_node_uid.gbl_next_uid = -1
```

Comment:
Why make this -1, when the class initializes it to 0?

Reply:
The new uid function call increments and returns it, so it had to be different